### PR TITLE
add mentor nathalier

### PR DESCRIPTION
**Pull request by nathalier**

### CODE REVIEW

File: mentors/python/n.json [hunk 1]
```diff
@@ -70,5 +70,13 @@
     "link_url": "https://nahua.dev",
     "avatar_url": null,
     "bio": "I am a self-taught programmer with a background in liberal arts. I work with Python professionally in the intersection between software development and deep learning. In my spare time, I also play with Go, Rust, and Swift."
+  },
+  {
+    "github_username": "nathalier",
+    "name": "Natalie R",
+    "link_text": null,
+    "link_url": " ",
```
Comment (rpottsoh) [inline mentors/python/n.json:78]: URL cannot be white space.  Should be `null` if you do not have a link you wish to share.

**Review by rpottsoh — CHANGES_REQUESTED** [review]

**Comment by SleeplessByte** [comment]
Pinging @nathalier

File: mentors/python/n.json (hunk 1 above)
Comment (DanielNoord) [inline mentors/python/n.json:78]:
```suggestion
    "link_url": null,
```

**Comment by BethanyG** [comment]
Pinging @nathalier -- this branch has merge conflicts and requested changes that need to be resolved.  

In the interest of cleaning out PRs from this repo that are older than 8 months, I'll wait a week to hear from you and then close this PR.  Let us know if you have any questions or issues.  Thanks!

**Comment by angelikatyborska** [comment]
Is this PR even still necessary? I thought adding mentors via those JSON files is deprecated in v3. It all happens via the platform now.

**Comment by BethanyG** [comment]
🤦🏽‍♀️   **_you are so right, @angelikatyborska!_**  Thank you for the reminder.  Closing.  😄

**Pull request closed without merging**